core/session.rb

```diff
@@ -216,6 +217,9 @@ def commands
     when 'install_product'
       command = InstallProduct.new(ARGV, self, $out)
       exit_code = command.execute
+    when 'install_repo'
```

Comment:
The command must be called setup_repo. Previous implementation was removed and we need to introduce a new one.